lib/sequenceserver/makeblastdb.rb

```diff
@@ -138,8 +138,8 @@ def determine_formatted_fastas
     end
 
     def get_format(path, type)
-      exts = Dir["#{path}.*"].map { |p| p.split('.').last }.sort
-      EXPECTED_EXTENSIONS[type][exts] || []
+      exts = Dir["#{path}.#{type[0]}*"].map { |p| p.split('.').last }.sort
+      EXPECTED_EXTENSIONS[type][exts]
```

Comment:
`|| []` in line 142 is important regardless of the change in line 141, so let's keep it.

Reply:
@yeban I see, I'll revert it then